mdoc/Mono.Documentation/Updater/AttributeParserContext.cs

```diff
@@ -88,5 +97,18 @@ private void ReadNullableAttribute()
             NullableReferenceTypeProvider nullableReferenceTypeProvider = new NullableReferenceTypeProvider(provider);
             nullableAttributeFlags = new ReadOnlyCollection<bool?>(nullableReferenceTypeProvider.GetNullableReferenceTypeFlags());
         }
+
+        private void ReadTupleElementNames()
+        {
+            if (provider != null && provider.HasCustomAttributes)
+            {
+                var tupleNamesAttr = provider.CustomAttributes.Where(attr => attr.AttributeType.FullName == Consts.TupleElementNamesAttribute).FirstOrDefault();
+                if (tupleNamesAttr != null)
+                {
+                    var constructorArgs = tupleNamesAttr.ConstructorArguments.FirstOrDefault().Value as CustomAttributeArgument[];
```

Comment:
Looks weird that why it always FirstOrDefault?

Reply:
Length of ConstructorArguments is always 1 in below log.
https://apidrop.visualstudio.com/6fb8cb7d-5623-420c-946b-ca74e63ac8ba/_apis/build/builds/280867/logs/12

Reply:
> Looks weird that why it always FirstOrDefault?

Length of ConstructorArguments is always 1 in below log.
https://apidrop.visualstudio.com/6fb8cb7d-5623-420c-946b-ca74e63ac8ba/_apis/build/builds/280867/logs/12